bin/src/command/requests.rs

```diff
@@ -388,12 +388,11 @@ impl GatheringTask for LoadStaticConfigTask {
     ) {
         let mut messages = vec![];
         for (worker_id, response) in self.gatherer.responses {
-            match response.status {
-                ResponseStatus::Ok => {}
-                ResponseStatus::Failure => {
+            match ResponseStatus::try_from(response.status) {
+                Ok(ResponseStatus::Failure) => {
                     messages.push(format!("worker {worker_id}: {}", response.message))
                 }
-                ResponseStatus::Processing => {}
+                Ok(ResponseStatus::Ok) | Ok(ResponseStatus::Processing) | Err(_) => {}
```

Comment:
We should at least log the error in order to know if something has occurred

Reply:
done, as with the others.